Source/monster.h

```diff
@@ -285,6 +285,19 @@ struct Monster { // note: missing field _mAFNum
 		return *type().data;
 	}
 
+	/**
+	 * @brief Returns monster's name
+	 * Internally it returns a name stored in global array of monsters' data.
+	 * @return Monster's name
+	 */
+	const char *name() const

```

Comment:
Return `string_view` here and remove `.data()` calls below and the explicit conversion to `string_view` elsewhere